### ISSUE TITLE
Connect forms with redux-form

### ISSUE THREAD

**Issue by minaorangina** [DRDD2016/native#47]
_From @minaorangina on January 5, 2017 12:57_

- [x] basic setup
- [x] client-side validation
- [x] server-side validation

_Copied from original issue: DRDD2016/sparkn2#62_

**Comment by minaorangina** [comment]
Followed [this stackoverflow post](http://stackoverflow.com/questions/39331420/cant-get-redux-form-6-0-2-working-with-react-native-0-32-0) to get an idea of how to extract the input values 

You pass a function to React Form's `handleSubmit`, which is passed to the `onPress` of your form's button.

```js
function mySubmitHandler (values, dispatch, props) {
  // do some stuff
}

const MyComponent = ({ handleSubmit }) => {
  <View>

   // ...
      <TouchableHighlight
        onPress={handleSubmit(mySubmitHandler)}
      >
        <Text>SIGN UP</Text>
      </TouchableHighlight>
  </View>
}

**Comment by minaorangina** [comment]
Added [synchronous form validation](http://redux-form.com/6.4.3/examples/syncValidation/). Note: the method for doing this will depend on the version of Redux Form being used.

**Comment by heron2014** [comment]
Resolved by #175
